core/v1/job.k

```diff
@@ -25,6 +24,9 @@ schema Job(WorkloadBase):
     }
     """
 
+    # The kind
+    kind:                       "job" = "job"
```

Comment:
what does "job" = "job" mean? You want to define a string and it's default value is "job"?